### PR TITLE
Add notes about file structure

### PR DESCRIPTION
**Pull request by njam**

### CODE REVIEW

**Comment by njam** [comment]
@tomaszdurka @fauvel please review

**Comment by tomaszdurka** [comment]
Shall we add any note about views ?

**Comment by tomaszdurka** [comment]
Oh you did! :)

File: README.md [hunk 1]
```diff
@@ -143,6 +143,46 @@ CM_Paging_Photo_User                           # All photos of a given user
 CM_Paging_User_Country                         # All users from a given country
 ```
 
+Structuring files
+-----------------
+Class definitions should be grouped by topic and stored in a common directory.
+Within such a *topic module* directories should be used to group files with common parent classes.
+A *topic module* can again contain a directory for another (sub) topic module.
+
+Example of a topic module "Payments", inside another topic module "Accounting":
```
Comment (fauvel) [inline README.md]: Wording: `with another topic module "Accounting" inside of it`

**Pull request merged**